BUILDING.md

```diff
@@ -22,6 +23,18 @@ and install them along with USD in a given directory.
 
 See instructions and examples in [README.md](README.md#getting-and-building-the-code).
 
+## Building With VCPKG
+
+You can download and install USD using the [vcpkg](https://github.com/Microsoft/vcpkg) dependency manager:
+```sh
+  git clone https://github.com/Microsoft/vcpkg.git
+  cd vcpkg
+  ./bootstrap-vcpkg.sh #.\bootstrap-vcpkg.bat(for windows)
+  ./vcpkg integrate install
+  ./vcpkg install usd
```

Comment:
Using the Microsoft Terminal, I got an error running this command:
```
Computing installation plan...
usd[core]:x86-windows is only supported on '!x86 & !arm'
```

I added `:x64-windows` to the command (so `./vcpkg install usd:x64-windows`) and that worked a lot better.  This step may be dependent on what OS you are installing it on, so may want to update it to reflect the different options.

Reply:
This is my mistake, I didn't notice that this port doesn't support x86.
Above step does not depend on the installed OS, the default triplets of vcpck are x86-windows.

I have updated the instruction.

Reply:
Thanks for pointing it out.